modules/viewing-drift-metrics.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: PROCEDURE
+
+[id="viewing-drift-metrics_{context}"]
+= Viewing data drift metrics for a model
```

Comment:
Should the title be file name and ID be changed to match the title here (or vice versa)?